exercises/scale-generator/example.go

```diff
@@ -0,0 +1,62 @@
+// Package scale provides a sample implementation
+package scale
+
+import "strings"
+
+var chromaticScale = []string{"C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A",
+	"A#", "B"}
+var flatChromaticScale = []string{"C", "Db", "D", "Eb", "E", "F", "Gb", "G", "Ab",
+	"A", "Bb", "B"}
+var flatKeys = []string{"F", "Bb", "Eb", "Ab", "Db", "Gb", "d", "g", "c", "f", "bb",
+	"eb"}
+
+// Scale returns a type of scale based on the inputs
+func Scale(tonic, interval string) []string {
+	if interval == "" {
+		interval = strings.Repeat("m", 12)
+	}
+	ft := formatTonic(tonic)
+	start := findStart(ft, chromaticScale)
+	scale := chromaticScale
+	if flatKey(tonic, flatKeys) {
+		start = findStart(ft, flatChromaticScale)
```

Comment:
Now you can also write this call one after `if` by using scale.

Reply:
I don't understand what you mean here

Reply:
since you set correct scale, you can call `findStart` once after `if` like

```go
if flatKey(tonic, flatKeys) {
  ...
}
start := findStart(ft, scale)
```

Reply:
ah, I see. That's good!